src/extensions/default/bramble/lib/ConsoleManager.js

```diff
@@ -15,8 +17,30 @@ define(function (require, exports, module) {
         var args = data.args;
         var type = data.type || "log";
 
-        // TODO: Show this in Custom Console UI, see issue #1675 in Thimble
+        if (type === "error-handler") {
+            var regex = new RegExp('(blob:.+):([^:]+):(.+)', 'gm'),
+                endingRegex = new RegExp(':([0-9]+):([0-9]+)$', 'gm'),
+                stackTrace = args["stack"].split("@"),
+                newArgs = [];
+            
+            // Handle Blob URLs
+            for(var i = 1; i < stackTrace.length; i++){
+                var stackItem = stackTrace[i].match(regex);
+                stackItem = (BlobUtils.getFilename(stackItem[0].replace(endingRegex, ''))).split('/');
+                newArgs.push('Function Line:' + stackItem[1] + ' File:' + stackItem[stackItem.length -1 ]);
+            }
+            
+            newArgs.splice(0, 0, args["messsage"]);
+            args = newArgs;
+            type = "error";
+        }
+
+        if (type === "time" || type === "timeEnd"){
+            args[0] = type + ": " + args[0];
+        }
+
         console[type].apply(console, args);
```

Comment:
Just for debugging, I can remove the `console[type].apply(console, args)` line since it's no longer needed.